src/Compilers/CSharp/Test/Emit/CodeGen/CodeGenTupleTest.cs

```diff
@@ -2994,13 +2995,14 @@ public static void Main()
         }
 
         [Fact]
-        public void DistinctTupleTypesInCompilationCannotAssign()
+        public void DistinctTupleTypesInCompilationCanAssign()
```

Comment:
I already mentioned (to please add a) LDM issue confirming that this is intended behavior.